tmt/cli.py

```diff
@@ -34,6 +35,24 @@
 # Explore available plugins (need to detect all supported methods first)
 tmt.plugins.explore(tmt.log.Logger.get_bootstrap_logger())
 
+
+class TmtExitCode(enum.IntEnum):
+    # "At least one test passed, there was no fail, warn or error."
```

Comment:
```suggestion
    # At least one test passed, there was no fail, warn or error.
```

No point for quotes?

Reply:
Converted to docstrings.